src/Metadata/MetadataFormatInterface.php

```diff
@@ -25,4 +25,9 @@ public function serialize(ImageMetadata $metadata, array $preserveKeys): string;
      * @throws InvalidImageMetadataException
      */
     public function parse(string $binaryChunk): array;
+
+    /**
+     * @return array<string,list<string>>
+     */
+    public function toReadable(array $data): array;
```

Comment:
What kind of array do I have to expect here? Maybe worth a comment in the interface?

Reply:
Done in 7c69e5962b56e74549325424062d03eee2c9b2be